docs/integration/nginx.md

```diff
@@ -1,12 +1,72 @@
 title: Nginx Monitoring Integration
 description: The Sematext monitoring agent can collect many metrics from NGINX instances, including requests, reading, writing, waiting, active and more. Utilize built-in anomaly detection, threshold, and heartbeat alerting and send notifications to email and various chatops messaging services. Correlate events & logs, filter metrics by server or time, and visualize your cluster's health with out of the box graphs and custom dashboards
 
-## Integration
+Sematext offers a simple Nginx monitoring agent, written entirely in Node.js without CPU and memory overhead. It's easy to install and require in your source code.
 
-- Agent: [https://github.com/sematext/sematext-agent-nginx](https://github.com/sematext/sematext-agent-nginx)
-- Instructions: [https://apps.sematext.com/ui/howto/Nginx/overview](https://apps.sematext.com/ui/howto/Nginx/overview)
+## Sematext Nginx Monitoring Agent Quick Start
```

Comment:
Maybe we can think about headings from the SEO perspective a bit?  Maybe "Sematext" is not needed here since these are obviously ST docs?  Maybe "Quick Start" is not needed?  Maybe "Install", as in "Install Nginx Monitoring Agent" would make sense?